lib-python/littleboxes/xword.py

```diff
@@ -1,22 +1,24 @@
 import logging
 import puz
 
+
```

Comment:
Ah I forgot about that!  It's reflex now - Eclipse has autopep8 built in as a code formatter, it does some nice indenting for you, keeps lines from being too long sometimes, and makes sure there aren't extra whitespace characters.  I can stop using it also, or make sure to undo when I accidently do, but it's pretty nice if you can find how to use it outside of eclipse/however you code